sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -407,80 +366,25 @@ export class MessageSender extends LinkEntity<AwaitableSender> {
   }
 
   /**
-   * Will reconnect the sender link if necessary.
+   * Closes the rhea link.
+   * To be called when connection is disconnected, onAmqpClose and onSessionClose events.
    * @param {AmqpError | Error} [senderError] The sender error if any.
    * @returns {Promise<void>} Promise<void>.
    */
   async onDetached(senderError?: AmqpError | Error): Promise<void> {
+    if (senderError) {
+      log.error(
+        "[%s] Closing the rhea link on the Sender '%s' with address " + "'%s': %O",
+        this._context.connectionId,
+        this.name,
+        this.address,
+        senderError
```

Comment:
Each called of `onDetached()` is logging the error that was seen. The `closeLink()` method does its own logging regarding the fact that it is closing the link. So, what does the above logging statement give us?

Reply:
Well, just additional logging.
No strong reasons to keep it.

Reply:
Removed.